modules/crop-ffmpeg/src/main/java/org/opencastproject/crop/impl/CropServiceImpl.java

```diff
@@ -280,7 +285,7 @@ private File cropFFmpeg(File mediaFile) throws CropException {
 
     if (cropValue > (widthVideo / maxCroppingRatio)) {
       logger.info("Black area in the video is considered too large for cropping. File: " + mediaFile.getAbsolutePath()
```

Comment:
Is something like this a thing we can detect with checkstyles?  It would be nice to require the use of the object parameters rather than allowing stringbuilding.

Reply:
Good question. A quick google search didn't show any built-in checkstyle rules for that, unfortunately. :/

Reply:
I didn't figure, but it was worth a shot :)